tests/CloudEventFunctionsWrapperTest.php

```diff
@@ -127,7 +127,7 @@ function (CloudEvent $foo, $bar = null) {
 
     public function testWithFullCloudEvent()
     {
-        self:$functionCalled = false;
+        self::$functionCalled = false;
```

Comment:
Is this test included in CI? Curious how this was a standing bug.

Reply:
Believe it or not, that's valid PHP. And not setting `$functionCalled` to `false` would not produce an error, only open a possibility for a false positive. 

I don't know why, but that's how it is. Anything before a single colon appears to be valid syntax:

```php
# this is all valid PHP for some reason
self:
foo:
asdfasdfgh:
;
```